backtester/config/examples/ftx-cash-carry.strat

```diff
@@ -36,8 +36,9 @@
    },
    "min-slippage-percent": "0",
    "max-slippage-percent": "0",
+   "taker-fee-override": "0.002",
```

Comment:
This is missing maker-fee-override.

Reply:
This is on purpose to highlight that it will retrieve the exchange's maker fee